### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by foolhardy21**
This PR has:
- Cart Page.

For responsiveness, I am using `window.resize` and `window.innerWidth` to add and remove classes. Let me know if you have a better idea. 

Feel free to review. Thanks.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+.cart-item {
+    background: var(--c-white);
+}
+
+.cart-item-image {
+    width: 35%;
+    max-width: 305px;
+    max-height: 305px;
+}
```
Comment (Hamza-777) [inline style.css:35]: Using rem would be nice here

**Review by Hamza-777 — APPROVED** [review]
Looks great overall.

**Comment by ShekharDhangar** [comment]
looks good

**Comment by foolhardy21** [comment]
> looks good

@ShekharDhangar . if you think it is good to merge then can you approve the code? a comment on conversation is not a valid review. thanks.

**Review by ShekharDhangar — APPROVED** [review]
looks good

**Review by Developers-codz — APPROVED** [review]
Looking Good to me. But I think instead of using window.innerWidth you can try doing it in css inside media query.

**Comment by foolhardy21** [comment]
> Looking Good to me. But I think instead of using window.innerWidth you can try doing it in css inside media query.

thanks for suggestion. yes i did this first but then writing it in media query would be redundant as I already have it as a util class in cl that can be added/removed from the script .

File: style.css (hunk 1 above)
Comment (foolhardy21) [reply style.css:35]: hey @Hamza-777 i have made the changes

**Pull request merged**